contracts/test/DaiEchidnaTest.sol

```diff
@@ -41,6 +47,8 @@ contract DaiEchidnaTest {
 
     /// @dev Test that supply and balance hold on burn
     function burn(uint256 wad) public {
+        wad = wad % uint256(-1);
```

Comment:
Similar thing here, you might try to `burn` more DAI than the holder has. So I might instead do something like `wad = 1+ (wad % holderBalance)`.

Reply:
Done.

Reply:
Done.